internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1490,6 +1500,7 @@ func resourceKubernetesClusterRead(d *pluginsdk.ResourceData, meta interface{})
 			d.Set("private_link_enabled", accessProfile.EnablePrivateCluster)
 			d.Set("private_cluster_enabled", accessProfile.EnablePrivateCluster)
 			d.Set("private_dns_zone_id", accessProfile.PrivateDNSZone)
+			d.Set("private_cluster_public_fqdn_enabled", accessProfile.EnablePrivateClusterPublicFQDN)
```

Comment:
(same here) - this'll be defaulted to `nil` based on previous experience with the AKS API when this is provisioned using an older API Version and then retrieved using a newer API version, so we likely need to see a default value for this

Reply:
It will be no changes, because it'll be `nil` in the response. But I think adding a default value makes sense to me.